src/contracts/proxies/Vault721.sol

```diff
@@ -2,15 +2,16 @@
 pragma solidity 0.8.19;
 
 import {ERC721} from '@openzeppelin/token/ERC721/ERC721.sol';
-import {ERC721Enumerable} from '@openzeppelin/token/ERC721/extensions/ERC721Enumerable.sol';
+import {ERC721EnumerableUpgradeable} from
+  '@openzeppelin-upgradeable/token/ERC721/extensions/ERC721EnumerableUpgradeable.sol';
 import {IODSafeManager} from '@interfaces/proxies/IODSafeManager.sol';
 import {ODProxy} from '@contracts/proxies/ODProxy.sol';
 import {NFTRenderer} from '@contracts/proxies/NFTRenderer.sol';
 
 // Open Dollar
 // Version 1.5.8
 
-contract Vault721 is ERC721Enumerable {
+contract Vault721 is ERC721EnumerableUpgradeable {
```

Comment:
I think it may be necessary to add a short comments explaining that the use of the "upgradeable" feature is limited only to the use of the initializer, and the contract is not actually upgradeable.

Please add this for all newly upgradeable contracts